src/open_inwoner/scss/components/File/File.scss

```diff
@@ -7,11 +7,35 @@
     display: flex;
   }
 
-  &__file > *[class*='icon']:first-child,
-  &__file > *[class*='Icon']:first-child {
-    background-color: var(--color-gray-light);
+  &__notification {
+    display: flex;
+    flex-direction: row;
```

Comment:
Now that you've wrapped this element correctly, you should give it a different line-height, conforming to the design:
`line-height: var(--spacing-medium);`